### DISCUSSION TITLE
抖一抖判断不出来问题

### DISCUSSION THREAD

**Discussion by Xiaoprinceprogrammer** [Mrs4s/go-cqhttp#1615] [Q&A] [answered]
{"post_type":"message","message_type":"private","time":1659595469,"self_id":2401858097,"sub_type":"friend","user_id":1317967909,"target_id":2401858097," message","raw_message":"","font":0,"sender":{"age":0,"nickname":"晓王子~"""""sex":"unknown", "user_id":1317967909},"message_id":-1505963132}
”你们看这是抖一抖的数据，哪有你们写的参数？

**Comment by Mrs4s** [comment, marked as answer]
抖一抖不支持
